Algorithm.Python/Alphas/ArtificiallyDepressedPricesAlpha.py

```diff
@@ -0,0 +1,190 @@
+# QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+# Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+''' 
+    Assets whose current trading volume is below its average volume and
+    whose price is below its recent average are sometiems viewed as being
+    priced artificially low, and as such will rebound in value once trading
+    volume picks up again. Using Simple Moving Average indicators, we check
+    for these conditions and generate insights acoordingly for the top 50
+    assets by Dollar Volume using a Coarse Universe selection function.
+
+    This alpha is part of the Benchmark Alpha Series created by QuantConnect which are open
+    sourced so the community and client funds can see an example of an alpha.
+'''
+
+from clr import AddReference
+AddReference("System")
+AddReference("QuantConnect.Algorithm")
+AddReference("QuantConnect.Common")
+
+from System import *
+from QuantConnect import *
+from QuantConnect.Indicators import *
+from QuantConnect.Algorithm import *
+from QuantConnect.Algorithm.Framework import *
+from QuantConnect.Algorithm.Framework.Risk import *
+from QuantConnect.Orders.Fees import ConstantFeeModel
+from QuantConnect.Algorithm.Framework.Alphas import *
+from QuantConnect.Algorithm.Framework.Selection import * 
+from QuantConnect.Algorithm.Framework.Execution import *
+from QuantConnect.Algorithm.Framework.Portfolio import PortfolioTarget, EqualWeightingPortfolioConstructionModel
+
+import numpy as np
+from datetime import timedelta, datetime
+
+
+class ArtificiallyDepressedPricesAlgorithm(QCAlgorithmFramework):
```

Comment:
The algorithm class name and the file name should share the same name: `ArtificiallyDepressedPricesAlpha`.